community/releases/release-manager-runbook/index.md

```diff
@@ -0,0 +1,329 @@
+
+    a. Select the oldest flow.
+
+    b. Click **Configure** from the left hand side of the menu.
+
+    c. Scroll to the **Build Triggers** section.
+
+    d. Cut text out of **Schedule** box (you will need to paste this soon).
+    
+    e. Save.
+
+6. Create a new Flow_BuildAndValidate_* flow for the release branch.
+
+    a. Click **New Item** on the left hand side of the main menu.
+
+    b. Set **Name** to Flow_BuildAndValidate_${RELEASE} (e.g., `1_18_x`). Note
+    that the newer versions of Jenkins seem to disallow the `.` character in the
+    job name, so will want to `s/./_`.
+    
+    c. Set **Copy from** to Flow_BuildAndValidate.
+
+    d. In the **Build Triggers** section, paste cut text from the second-oldest
+    flow.
+    
+    e. Set **GITHUB_REPO BRANCH** to `${RELEASE_BRANCH}`.
+    
+    f. Set **PROCESS_GITHUB_REPO_BRANCH** TO `${RELEASE_BRANCH}`.
+    
+7. At this point, there should exist the following Flow_BuildAndValidate_* jobs:
+
+    a. Flow_BuildAndValidate_${RELEASE-3} (DEACTIVATED)
+    
+    b. Flow_BuildAndValidate_${RELEASE-2} (BUILDING NIGHTLY)
+    
+    c. Flow_BuildAndValidate_${RELEASE-1} (BUILDING NIGHTLY)
+    
+    d. Flow_BuildAndValidate_${RELEASE} (BUILDING NIGHTLY)
+    
+    e. Flow_BuildAndValidate (master, BUILDING NIGHTLY)
+    
+8. Run the Flow_BuildAndValidate_${RELEASE} job.
+
+    a. Select `stable` for **HALYARD_RELEASE_TRACK**.
+    
+    b. This will automatically update the [changelog gist](https://gist.github.com/spinnaker-release/4f8cd09490870ae9ebf78be3be1763ee)
+    on GitHub.
+    
+9. Add the new Flow_BuildAndValidate_${RELEASE} job to the public
+[Build Statuses page](https://www.spinnaker.io/community/contributing/build-statuses/#nightly-and-release-integration-tests).
+Ping #dev with some version of this message, including a link to the correct
+section of the changelog gist:
+
+> The release window for Spinnaker $VERSION is now open!  This means that
+> release branches have been cut from master and those branches are only
+> accepting fixes for existing features.  Please contact $YOUR_NAME
+> (slack: $YOUR_SLACK_ID, github: $YOUR_GITHUB_ID, or email: $YOUR_EMAIL) if you
+> would like a fix cherry-picked into the release or you would like to highlight
+> a specific fix or feature in the release’s changelog. If you’d like to jog
+> your memory of everything to be released with Spinnaker $VERSION, see the raw
+> changelog here: $LINK_TO_CHANGELOG.
+
+10. Share a curated changelog with any partners from the community who wants to
+add notes.
+
+11. When the Flow_BuildAndValidate_${RELEASE} job passes, ping #dev with a
+message that release candidate is now validated and can be tested by running:
+
+```
+hal config version edit --version ${RELEASE_BRANCH}-latest-validated
+```
+
+## Second Monday of the release window
+
+1. Check for any PRs waiting to be [cherry-picked](https://github.com/search?q=org%3Aspinnaker+label%3Acherry-pick+state%3Aopen+type%3Apr).
```

Comment:
I usually use something like [this](https://github.com/pulls?utf8=%E2%9C%93&q=org%3Aspinnaker+is%3Apr+is%3Aopen+-base%3Amaster) because it works even if the `cherry-pick` label isn't applied.